apps/site/components/EOL/Alert.tsx

```diff
@@ -0,0 +1,18 @@
+import AlertBox from '@node-core/ui-components/Common/AlertBox';
+import { useTranslations } from 'next-intl';
+
+import Link from '#site/components/Link';
+
+const EOLAlert = () => {
+  const t = useTranslations('components.eolAlert');
```

Comment:
Usually we don't do shorthands, we alays call t() with the full translation path to make it easier to statically analyze and find usages.

Reply:
(This applies to call `useTranslations()` calls on your PR.